src/converter/entities.ts

```diff
@@ -52,10 +52,14 @@ export const convertComputer = (
         networkAddresses: undefined,
         macAddress: data.network_addresses
```

Comment:
Maybe we should extract this into a separate utility function and add unit tests? It would be helpful to see a sample input and expected output when reviewing the code.